README.md

```diff
@@ -31,6 +31,7 @@ A **pure** _(in both senses of the word!)_ **Scala 3** logging library with **no
 | core   | ✅   | ✅        | ✅      |
 | http4s | ✅   | ✅        | ✅      |
 | slf4j  | ✅   | 🚫        | 🚫      |
```

Comment:
Should we format the table above nicely?

Reply:
Yes, there's this annoying issue where it tries to format the code snippets when inside markdown and it just _destroys_ it. I'll do it by hand